gradle-versions-plugin/build.gradle

```diff
@@ -41,14 +50,16 @@ dependencies {
   testImplementation 'org.spockframework:spock-junit4:2.1-M2-groovy-3.0', { exclude module: 'groovy-all' } // Use localGroovy()
 }
 
-task docsJar(type: Jar, dependsOn: dokkaHtml) {
+tasks.register('docsJar', Jar) {
+  dependsOn dokkaHtml
   group = 'Publications'
   description = 'Create jar of documentation.'
   archiveClassifier = 'javadoc'
   from dokkaJavadoc.outputDirectory
 }
 
-task sourcesJar(type: Jar, dependsOn: classes) {
+tasks.register('sourcesJar', Jar) {
+  dependsOn classes
```

Comment:
I think nowadays one can use `java.withSourcesJar()`

Reply:
I can move us to use https://github.com/vanniktech/gradle-maven-publish-plugin to help remove these custom tasks in the future.

Reply:
@ben-manes Actually, we are only using Gradle's publish plugin - https://docs.gradle.org/current/userguide/publishing_gradle_plugins.html we are not publishing the docs or sources. Looks like we have not published to maven in a long time? https://mvnrepository.com/artifact/com.github.ben-manes/gradle-versions-plugin

Reply:
Oh right, it predated the plugin portal by a few years.